modules/integration/tests-integration/tests-mediator-1/src/test/java/org/wso2/carbon/esb/mediator/test/payload/factory/JsonFormat_IncomingJson_CTXJsonValues_WithStream_TestCase.java

```diff
@@ -0,0 +1,113 @@
+package org.wso2.carbon.esb.mediator.test.payload.factory;
+
+
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.carbon.automation.engine.annotations.ExecutionEnvironment;
+import org.wso2.carbon.automation.engine.annotations.SetEnvironment;
+import org.wso2.carbon.automation.test.utils.http.client.HttpURLConnectionClient;
+import org.wso2.esb.integration.common.utils.ESBIntegrationTest;
+
+import java.io.Reader;
+import java.io.StringReader;
+import java.io.StringWriter;
+import java.io.Writer;
+import java.net.URL;
+
+import static org.testng.Assert.assertTrue;
+
+public class JsonFormat_IncomingJson_CTXJsonValues_WithStream_TestCase extends ESBIntegrationTest{
```

Comment:
Better rename class name with camel case. We usually don't use _ in class/method names.